packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -1,50 +1,53 @@
 import { ISemanticTextColors } from './ISemanticTextColors';
 
+// WARNING: The comment below must use valid markdown, or it will break the website.
```

Comment:
This file wasn't valid markdown, and starting the headings at h2 didn't work in the context of the website.